packages/core/tsconfig.json

```diff
@@ -2,13 +2,10 @@
   "extends": "../../tsconfig.json",
   "compilerOptions": {
     "declaration": true,
-    "module": "CommonJS",
```

Comment:
Justin set these up and they were working.  Are we positive the package will still be compatible with current usage based on these changes? (ie "module", "target" and "rootDir" changing)